rrmngmnt/filesystem.py

```diff
@@ -161,3 +162,59 @@ def wget(self, url, output_file, progress_handler=None):
                 "Failed to download file from url {0}".format(url)
             )
         return output_file
+
+    def get_file_permissions(self, file_path):
+        """
+        Get file permissions
+
+        :param file_path: absolute path to file
+        :type file_path: str
+        :return: file permission in octal form(example 644)
+        :rtype: str
+        """
+        cmd = ["stat", "-c", "\"%a\"", file_path]
+        return self._exec_command(cmd=cmd)
+
+    def get_file_owner(self, file_path):
+        """
+        Get file user and group owner
+
+        :param file_path: absolute path to file
+        :type file_path: str
+        :return: file user and group owner names(example ['root', 'root'])
+        :rtype: list
+        """
+        cmd = ["stat", "-c", "\"%U %G\"", file_path]
+        return self._exec_command(cmd=cmd).split()
+
+    def is_user_exist(self, user_name):
```

Comment:
I am not sure whether these methods belongs to this module, it actually doesn't have nothing to do with filesystem